test/.gitignore

```
@@ -1 +1,9 @@
-output/
+output/checkpoints/
+output/results/STUBP02/
+output/results/STUBP03/
+output/results/STUBP04/
+output/results/STUBP05/
+output/results/STUBP06/
+output/results/STUBP07/
+output/results/STUBP08/
+output/results/merged/
```

I would leave this as it was, and maybe use `git add --force` for the individual files for this use case.

```suggestion
output/
```

Alternatively, we can be more succinct with:
```
/output
/output/results/*
!/output/results/STUBP01
```
 I actually think I like the former better... it freezes the files in their current state and prevents accidental modification.